### PR TITLE
Super basic style

### PR DESCRIPTION
**Pull request by SallyMcGrath**
- very minor html changes
- there's no point doing loads as we will reactify the thing
- super basic style thrown on
- moved the TOC off the view  - header footer menu and some buttons
- none of this is really ok but it's a little easier to parse, maybe
- a tiny improvement, I offer

### CODE REVIEW

File: exercises.js [hunk 1]
```diff
@@ -39,9 +39,10 @@ function findExercises() {
 
 function setupToc(exercises) {
   let $toc = document.getElementById("toc");
-  let $ul = document.createElement("ul");
+  let $ul = document.createElement("ol");
```
Comment (gregdyke) [inline exercises.js:42]: rename variable

File: exercises.js [hunk 2]
```diff
@@ -55,7 +56,7 @@ function setupToc(exercises) {
 function addDomElements(exercises) {
   for (let i = 0; i < exercises.length; i++) {
     let exercise = exercises[i];
-    let $nav = document.createElement("div");
+    let $nav = document.createElement("nav");
```
Comment (gregdyke) [inline exercises.js:59]: there's a nav html element??? who knew!!? (not me anyway)

File: index.html [hunk 3]
```diff
@@ -1481,16 +1401,16 @@ <h2>Some more "projects"</h2>
...
-        background-color: var(--brand-red); color:white; padding: 40px 20px
-      }
-      .c-site-footer a { color: var(--brand-white); text-decoration: none; border-bottom: 2px solid var(--brand-black); padding:2px}
-    </style>
-    <footer class="c-site-footer">
+    <footer>
       <h3>CYF Blocks is a project of <a href="https://codeyourfuture.io">Code Your Future</a></h3>
       <p>Recap your HTML/CSS knowledge with this <a href="https://scrimba.com/learn/htmlcss">free crash course</a> on Scrimba.</p>
     </footer>
+     <!-- <dialog aria-live="assertive" class="toast" id="toast" aria-labelledby="toast-title" closed>
```
Comment (gregdyke) [inline index.html:1408]: toast?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+.button:hover,
+.button:focus {
+  background: var(--theme-color--ink);
+  color: var(--theme-color--paper);
+}
+button:active {
+  background: var(--theme-color--pop);
+  color: var(--theme-color--paper);
+}
+:focus {
```
Comment (gregdyke) [inline style.css:216]: this seems to also affect blockly blocks themselves. Not to mention leaving the last button pressed with a weird outline

**Review by gregdyke — CHANGES_REQUESTED** [review]
Great start while waiting for React!

- Could we have less horizontal padding - it's losing some valuable blocks screen estate (at least on my not very big screen)
- Could we have a bit of vertical padding/margin around the actions buttons? They particularly get crowded by the output area
- "Exercises menu" (or something else more descriptive) rather than just "menu"?
- I think the focus/button outline needs a bit of tinkering. Leaving a button selected after it's clicked is a bit weird and it affects the input areas in blockly

Some kind of "copied to clipboard" popup is really missing after pressing share (I think it becomes a must have before the first course, but not needed as part of this change)

File: exercises.js (hunk 2 above)
Comment (SallyMcGrath) [reply exercises.js:59]: BTW almost none of this html will remain in the React version as this DOM just doesn't make sense as is. But let's do what we can for now!

File: index.html (hunk 3 above)
Comment (SallyMcGrath) [reply index.html:1408]: If I get 20 minutes it will manifest there.

File: style.css (hunk 4 above)
Comment (SallyMcGrath) [reply style.css]: sorted

File: index.html (hunk 3 above)
Comment (SallyMcGrath) [reply index.html]: toast

**Review by gregdyke — CHANGES_REQUESTED** [review]
Amazing! Thanks for all the tweaks.

There is a small bug left: when I open the menu and select an exercise, the menu doesn't close (but the exercise is selected)

**Comment by SallyMcGrath** [comment]
> Amazing! Thanks for all the tweaks.
> 
> There is a small bug left: when I open the menu and select an exercise, the menu doesn't close (but the exercise is selected)

fixed in this tiny commit

I also moved the buttons left

**Comment by gregdyke** [comment]
I don't see flash or code for flash, but maybe I'm confused - I've had a frustrating day so I'm not at my brightest :)

**Review by gregdyke — APPROVED** [review]
wheee!

**Pull request merged**